### DISCUSSION TITLE
JS for GUI

### DISCUSSION THREAD

**Discussion by XeniaMonika** [closingthegap/closingthegap.github.io#128] [General]
I would like to contribute more to the GUI and to be able to work with it more independently. For example, in case we change the scheme, the GUI form has to be changed as well and it would be easier if I could do it directly. So my question is  if I could use some of my 20 hours to learn JS

**Comment by theodore-s-beers** [comment]
Sounds like a good idea. I'm not sure who, at the moment, technically has the authority to approve such things.

**Comment by XeniaMonika** [comment]
Cool! Thank you. I don´t know it either...

**Comment by Jomula** [comment]
I'd say just do it.

**Comment by theodore-s-beers** [comment]
Now I think it's safe to say, definitely go for it.

**Comment by XeniaMonika** [comment]
Haha! Thanks! Now I will continue my coursera course with a clear conscience ;)
